src/tests/class_tests/openms/source/AbsoluteQuantitationStandards_test.cpp

```diff
@@ -28,38 +28,70 @@
 // ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 // --------------------------------------------------------------------------
-// $Maintainer: Douglas McCloskey $
-// $Authors: Douglas McCloskey $
+// $Maintainer: Douglas McCloskey, Pasquale Domenico Colaianni $
+// $Authors: Douglas McCloskey, Pasquale Domenico Colaianni $
 // --------------------------------------------------------------------------
-//
 
 #include <OpenMS/CONCEPT/ClassTest.h>
 #include <OpenMS/test_config.h>
 
 ///////////////////////////
-
 #include <OpenMS/METADATA/AbsoluteQuantitationStandards.h>
+#include <OpenMS/FORMAT/AbsoluteQuantitationStandardsFile.h>
+#include <OpenMS/FORMAT/FeatureXMLFile.h>
+///////////////////////////
 
 using namespace OpenMS;
 using namespace std;
 
-///////////////////////////
-
 START_TEST(AbsoluteQuantitationStandards, "$Id$")
 
 /////////////////////////////////////////////////////////////
 
-AbsoluteQuantitationStandards* ptr = nullptr;
-AbsoluteQuantitationStandards* nullPointer = nullptr;
-START_SECTION((AbsoluteQuantitationStandards()))
-	ptr = new AbsoluteQuantitationStandards();
-	TEST_NOT_EQUAL(ptr, nullPointer)
+AbsoluteQuantitationStandards* ptr = 0;
+AbsoluteQuantitationStandards* null_ptr = 0;
+const String calib_conc_path = OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationStandardsFile_150516_calibration_concentrations_cut.csv");
```

Comment:
Instead of loading these files into the test, it might be better just to hard code the parts that you need for testing in the code to reduce the footprint in the codebase.